pallets/file-system/src/types.rs

```diff
@@ -61,6 +62,24 @@ pub struct StorageRequestMetadata<T: Config> {
     pub bsps_volunteered: T::StorageRequestBspsRequiredType,
 }
 
+impl<T: Config> StorageRequestMetadata<T> {
+    pub fn to_file_metadata(
+        self,
+    ) -> FileMetadata<
+        { shp_file_key_verifier::consts::H_LENGTH },
+        { shp_file_key_verifier::consts::FILE_CHUNK_SIZE },
+        { shp_file_key_verifier::consts::FILE_SIZE_TO_CHALLENGES },
```

Comment:
FYI, I'm in the process of removing this type from here, but for this PR is perfectly fine.